pkg/apis/admissionregistration/v1/register.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2020 Authors of Arktos.
+
```

Comment:
Missing kubernetes copyright

Reply:
This is by design for current arktos_copyright.sh. all new added files only has arktos copyrights if we didn't  add new added files to hack/arktos_copyright_copied_k8s_files.